tests/PhpSpreadsheetTests/Reader/XEEValidatorTest.php

```diff
@@ -2,12 +2,35 @@
 
 namespace PhpOffice\PhpSpreadsheetTests\Reader;
 
+use PhpOffice\PhpSpreadsheet\Cell\DataType;
 use PhpOffice\PhpSpreadsheet\Reader\BaseReader;
+use PhpOffice\PhpSpreadsheet\Spreadsheet;
 use PHPUnit_Framework_TestCase;
 
 class XEEValidatorTest extends PHPUnit_Framework_TestCase
 {
     /**
+     * @var Spreadsheet
+     */
+    private $spreadsheetXEETest;
+
+    /**
+     * @return Spreadsheet
+     */
+    protected function loadXEETestFile()
+    {
+        if (!$this->spreadsheetXEETest) {
+            $filename = __DIR__ . '/../../../samples/templates/Excel2003XMLTest.xml';
```

Comment:
because we `chdir(__DIR__);` in `tests/bootstrap.php`, you can simplify this line with:

```php
$filename = '../samples/templates/Excel2003XMLTest.xml';
```